WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -1,77 +1,79 @@
 <% provide(:title, "Registrations for #{@competition.name}") %>
 
 <%= render layout: "nav" do %>
-  <%= wca_table data: { toggle: "table", sort_name: "name" } do %>
-    <thead>
-      <tr>
-        <th class="name" data-sortable="true" data-field="name">Name</th>
-        <th class="wca-id" data-sortable="true">WCA ID</th>
-        <th class="country" data-sortable="true">Citizen of</th>
+  <% cache ["registrations_index", @registrations.pluck(:id), @registrations.maximum(:updated_at)] do %>
```

Comment:
1. In the controller we sort registrations by name so I think it's ok to just pluck the ids (Don't I get your idea?)
2. That's right!